.github/workflows/release-arm.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
           MODEL: ${{ matrix.model }}
         run: |
           mkdir sarif
-          sudo mv release/*.sarif sarif/
+          sudo mv build/*.sarif sarif/
```

Comment:
doesn't `release.yaml` also need the same fix?

Reply:
weirdly enough no, it seems to work in that job: https://github.com/kairos-io/kairos/actions/runs/4532595843/jobs/8008458849

Reply:
oh I see, the release job moves the files from /build to /release....

Reply:
not sure why on arm we dont directly do that or why we move the files into release/ and not work with the build directory directly. Probably due to permission stuff?